unable to expand properties within a dict

My issue's symptoms sound similar to #83 , but for python. I can inspect and expand variables, including lists and dicts, but if I try to expand anything within a dict, I get a `NameError`
For example, I can expand `user_pool_config.properties` (dict) and I see that `user_pool_config.properties['AutoVerifiedAttributes']` is a list with 1 element, but when I try to expand that, I get `- [ERROR] NameError: name 'AutoVerifiedAttributes' is not defined`. However, if I select `user_pool_config.properties['AutoVerifiedAttributes']` and run `<leader>e` it shows the value (which is, in turn, not expandable either)

I have patched my dbgp/client.py as suggested in :help. It looks like I'm at revision 4c6a7ca of vdebug

--- a/pythonx/vdebug/dbgp.py
+++ b/pythonx/vdebug/dbgp.py
@@ -360,8 +360,11 @@ class Api:
     def property_get(self, name):
         """Get a property.
         """
-        return self.send_cmd('property_get', '-n %s -d 0' % name,
-                             ContextGetResponse)
+        return self.send_cmd(
+            'property_get',
+            '-n "%s" -d 0' % name.replace("\\", "\\\\").replace("\"", "\\\""),
+            ContextGetResponse
+        )
 
     def detach(self):
         """Tell the debugger to detach itself from this

